app/_locales/en/messages.json

```diff
@@ -3673,6 +3673,10 @@
     "message": "View your public key for $1.",
     "description": "The description for the `snap_getBip32PublicKey` permission. $1 is a name for the derivation path, e.g., 'Ethereum accounts'."
   },
+  "permission_walletSwitchEthereumChain": {
+    "message": "Switch to and use the following network",
```

Comment:
Is there a distinction between "switching to" and "using" a network?
```suggestion
    "message": "Switch to the following network",
```

Reply:
> Is there a distinction between "switching to" and "using" a network?
Not in the way we're designing it. I could see this either way. This was a placeholder anyways, since this PR doesn't aim to implement the full UI. The current designs are actually:
![Screenshot 2024-05-15 at 4 11 58 PM](https://github.com/MetaMask/metamask-extension/assets/34557516/05cb5a4e-626d-4b9a-907f-8d0445def686)

https://www.figma.com/design/BBu3ry2zLZYOmyLOEhfWYB/Multichain-v2%2B---Extension?node-id=3140-44086&t=NahZKgd0W2vB9EVC-0